docs/tutorials.rst

```diff
@@ -15,22 +15,28 @@
 Tutorials
 =========
 
-The following tutorials demonstrate how to use Trieste to solve various optimization problems, and detail the functionality available in Trieste.
+Example optimization problems
+-----------------------------
 
-.. toctree::
-   :caption: Introductory
-   :maxdepth: 1
-
-   notebooks/introduction
-   notebooks/thompson_sampling
+The following tutorials explore various optimization problems using Trieste.
 
 .. toctree::
-   :caption: Advanced
    :maxdepth: 1
 
+   notebooks/expected_improvement
+   notebooks/thompson_sampling
    notebooks/inequality_constraints
    notebooks/failure_ego
 
+Trieste functionality in detail
+-------------------------------
+
+The following tutorials (or sections thereof) demonstrate how to use and extend specific Trieste functionality.
+
+* :doc:`Set up a basic Bayesian optimization routine<notebooks/expected_improvement>`
+* :ref:`Customize the acquisition function<notebooks/failure_ego:Create a custom acquisition function>`
+* :ref:`Customize the model optimization routine<notebooks/failure_ego:Create a custom optimize method>`
```

Comment:
this is what I meant by the hack. The first list uses `toctree`, the second a bulleted list. I can't see any difference from the user's perspective

Reply:
We have two kinds of things listed here: use ("set up a basic BO routine") and extend (customize ...).  Should we keep it like that or have two lists?

As for the title, do you like "How to..."? otherwise nothing cleverer comes to mind.

We could have other bullets in the "use" list: "using several models", "creating a complex observer". Not 100% sure it's a good idea though.

Reply:
One list for now I reckon. There's not enough for two lists, I think it would make it harder to read

We can add more items as we go. We've now done the hard part of working out how to add it.